### ISSUE TITLE
Merge state values into one column

### ISSUE THREAD

**Issue by edtro** [edtro/EdTro.AzureDevOps.Extensions.querybasedboards#24]
Like as mentioned in the question of Daniel Stevenson:

e.g. Task State = New, Bug State = Active
Show both in a Column called New

**Comment by schiltzj** [comment]
Hi - if we have custom state values would there be a way to add them to a given column?
Example, we have analysis, dev, and testing states such as "Gathering Requirements", "Development in Progress", "Testing In Progress", etc.

**Comment by edtro** [comment]
@schiltzj 
> Hi - if we have custom state values would there be a way to add them to a given column?
> Example, we have analysis, dev, and testing states such as "Gathering Requirements", "Development in Progress", "Testing In Progress", etc.

Hi, Thank you for responding.... but I am little bit confused about why you are responding on an item called 'Merge state values into one column'. 

So to clearify your question:
- Do you want to make certain that your custom states will show up as seperate columns on the board? Yes, I am checking the work item types to see what states are setup and will show them automatically. You can also do some advanced configuration; see: https://github.com/edtro/EdTro.AzureDevOps.Extensions.querybasedboards/blob/master/public/config.md (so you can e.g. modify the column header or show/hide the actual columns)
- Do you want to merge multiple states (including custom states) to one column? Well you can see that this is still an 'open' issue and I have not started development yet. Because I am alone and doing this project in my free spare time I cannot give any timetable or even guarantees that I am actually going to implement this new feature. However when there is some news regarding this, I will be updating this issue (like posting screenshots of early development etc).

Hopefully this will answer your question.

Kr
Edward

**Comment by schiltzj** [comment]
> Hopefully this will answer your question.

Yes; thank you. It was primarily to understand if custom values would be included. Thank you for the reference to the config.md as well.

**Comment by Shailja-19** [comment]
Hi, 

By when can we get this feature implemented?
